fetch/api/basic/scheme-data.any.js

```diff
@@ -3,7 +3,7 @@
 
 function checkFetchResponse(url, data, mime, fetchMode, method) {
   var cut = (url.length >= 40) ? "[...]" : "";
-  desc = "Fetching " + (method ? "[" + method + "] " : "") + url.substring(0, 40) + cut + " is OK";
+  let desc = "Fetching " + (method ? "[" + method + "] " : "") + url.substring(0, 40) + cut + " is OK";
```

Comment:
```suggestion
  var desc = "Fetching " + (method ? "[" + method + "] " : "") + url.substring(0, 40) + cut + " is OK";
```

To match the previous and following line. Changing everything to const/let would be fine too, but a side-by-side mix would be nice to avoid.